### DISCUSSION TITLE
Logo and OSD resources suggestions

### DISCUSSION THREAD

**Discussion by zhen-zen** [zhen-zen/YogaSMC#86] [General]
Some icons like Bluetooth on / off will also need separate resources.

Current resources:
- [AirplaneMode](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/AirplaneMode.pdf) - [Antenna](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Antenna.pdf)
- [Mic](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Mic.pdf) - [MicOff](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/MicOff.pdf)
- [Keyboard](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Keyboard.pdf) - [KeyboardOff](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/KeyboardOff.pdf)
- [Wifi](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Wifi.pdf) - [WifiOff](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/WifiOff.pdf)
- [BacklightHigh](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/BacklightHigh.pdf) - [BacklightLow](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/BacklightLow.pdf) - [BacklightOff](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/BacklightOff.pdf)
- [CapslockOn](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/CapslockOn.pdf) - [CapslockOff](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/CapslockOff.pdf)
- [Bluetooth](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Bluetooth.pdf)
- [Camera](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Camera.pdf)
- [FunctionKeyOn](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/FunctionKeyOn.pdf) - [FunctionKeyOff](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/FunctionKeyOff.pdf)
- [SecondDisplay](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/SecondDisplay.pdf)
- [Sleep](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Sleep.pdf)
- [Star](https://github.com/zhen-zen/YogaSMC/blob/master/YogaSMCNC/Resources/Star.pdf)

Not implemented:
- BT On / Off and discovery
- Yoga mode: laptop / tablet / stand / tent / stand_tent / flat

**Comment by Ab2774** [comment]
Hi, I don't have any problem with the icons as they're pretty similar to Apple's whatsoever, the only issue is that they're more bolder than Apple ones, maybe redesign them?
Thanks

**Comment by hexart** [comment]
[HUDICONS.zip](https://github.com/zhen-zen/YogaSMC/files/5677501/HUDICONS.zip)
I don't have yoga book, so I can't test yogasmc on my laptop. This is the source file of hud icons made by illustrator, pdf file can also opened by it if you need some modifications.

**Reply by zhen-zen** [reply]
Many thanks! Those icons are fancy! I will do some test and update the resources. However, there's already icons for keyboard backlight at `/System/Library/CoreServices/OSDUIHelper.app/Contents/Resources/{kBright,kBrightOff}.pdf`. So currently I will stick to that one.

And this driver is not restricted to Yoga series lol. Feel free to try it out!

**Reply by hexart** [reply]
Glad I can help! For thinkpad, it has two steps of keyboard backlight phase, so I drew them separately.

**Comment by junaedahmed** [comment]
Hello @zhen-zen ,

@hexart previously contributed some pretty cool logo and HUD icons for ThinkpadAssistant app. So I asked him to help this project also. He has a kind heart and responded to my request. If we can specify exact graphics need I think he can deliver best looking graphics for this project.

Regards
Junaed

**Reply by zhen-zen** [reply]
Sure, I will get a table for current icon status here.
